src/undomanager.js

```diff
@@ -22,21 +24,20 @@ define([
         $('.fd-undo-container').append(UNDO_ALERT);
     }
 
-    function toggleAlert(undoStack, vellum) {
+    function toggleAlert(undoStack) {
         if (undoStack.length && !alertShown()) {
             createAlert();
         } else if (undoStack.length === 0 && alertShown()) {
             $('.fd-undo-delete').remove();
         }
-        if (vellum) {
-            vellum.adjustToWindow();
-        }
     }
 
     function UndoManager(form) {
         var _this = this;
         _this.undoStack = [];
-        _this.vellum = form.vellum;
+        _this.form = form;
+
+        util.eventuality(this);
```

Comment:
I see what you mean. My thinking was basically that undoing a question removal is fairly similar to adding a question, which is handled by the form. But an undo manager could (maybe) handle events that aren't so form-related, and the actual `undo-reset` event is about the undo queue, not about the form. I think I'll rearrange this code.